docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile

```diff
@@ -20,7 +20,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install initramfs-tools libnuma-de
         wget ${ROCM_INSTALLER_REPO} && \
         sudo apt-get install -y ./${ROCM_INSTALLER_PACKAGE} && \
         sudo apt-get update -y && \
-        sudo amdgpu-install -y --usecase=graphics,rocm
+        sudo amdgpu-install -y --usecase=graphics,rocm --no-32
```

Comment:
probably we can use --usecase=multimedia,rocm instead?

Reply:
@rrawther The usecase multimediasdk was not tested thoroughly on rocm6.0 and these dockers are currently on that version of rocm. We can switch to multimediasdk in the next version